VaticanBAV/et/BAVet132.xml

```diff
@@ -0,0 +1,424 @@
+                            ለማኅደር፡ እስከ<supplied reason="omitted">፡</supplied> ዓርያም። በእንተ፡ <choice><corr>ተሠግዎቱ፡</corr><sic>ተግወቱ፡</sic></choice> ለወልደ፡ አምላክ፡ እምኔኪ፨
+                          </explicit>
+                        </msItem>
+                      </msItem>
+                    </msContents>
+                  
+                    <physDesc>
+                      <objectDesc form="Codex">
+                        <supportDesc>
+                          <support>
+                            <material key="parchment">
+                            </material>
+                          </support>
+
+                          <extent>
+                            <measure unit="leaf" n="128">128</measure>
+                            <measure unit="leaf" type="blank">1</measure>
+                            <locus from="1r" to="1v"/>
+                            <dimensions type="outer" unit="mm">
+                              <height>193</height>
+                              <width>125</width>
+                            </dimensions>
+                            </extent>
+                          
+                            <collation>
+                              <note>According to the catalogue, the manuscript consists of 19 quires and one 
+                              protective bifolio. The third, twelfth and penultimate quire are quaternions, the 
+                              fourth quire has 9 folios and the last quire has 5 folios. The other quires are
+                              quinions. This description does not seem to add up to 128 folios.</note>
+                              <signatures>No quire marks.</signatures>
+                              <!--constat fasciculis XIX sine notis, bifolio pro custodia prae-
+                              misso. Plerique fasciculi quiniones, tertius tamen, duodecimus et paenultimus quater-
+                              niones sunt, quartus 9 ff. habet ultimusque 5.-->
+        <!--    <list>
+              <item xml:id="q1" n="A">
+                <dim unit="leaf">2</dim>
+                <locus from="1" to="2"/>
+              </item>
+                                <item xml:id="q2" n="1">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="3" to="12"/>
+                                </item>
+                                <item xml:id="q3" n="2">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="13" to="22"/>
+                                </item>
+                                <item xml:id="q4" n="3">
+                                  <dim unit="leaf">8</dim>
+                                  <locus from="23" to="30"/>
+                                </item>
+                                <item xml:id="q5" n="4">
+                                  <dim unit="leaf">9</dim>
```

Comment:
the app won't be able to process it unless a s.l. is mentioned

Reply:
I know, but what is the convention for cases in which we don't know which leaf is the single one?

Reply:
we can write something (like for the middle, s.l. 5 no stub) and leave a comment that we have not checked